src/scala/microsoft-spark-2.3.x/src/main/scala/org/apache/spark/api/dotnet/DotnetBackendHandler.scala

```diff
@@ -168,13 +182,15 @@ class DotnetBackendHandler(server: DotnetBackend)
           case Some(jObj) => jObj.getClass.getName
           case None => "NullObject"
         }
-        val argsStr = args.map(arg => {
-          if (arg != null) {
-            s"[Type=${arg.getClass.getCanonicalName}, Value: $arg]"
-          } else {
-            "[Value: NULL]"
-          }
-        }).mkString(", ")
+        val argsStr = args
+          .map(arg => {
+            if (arg != null) {
+              s"[Type=${arg.getClass.getCanonicalName}, Value: $arg]"
+            } else {
+              "[Value: NULL]"
+            }
+          })
+          .mkString(", ")
```

Comment:
Was this change made only for aesthetic reasons?

Reply:
I'm not sure why, but scalafmt automatically made this change.